opendocdb-cts/internal/mongosh/testsuite.go

```diff
@@ -26,6 +26,7 @@ import (
 
 // ConvertRequest converts wirebson's request Document to a mongosh's JavaScript `runCommand`.
 func ConvertRequest(req *wirebson.Document) (string, error) {
+	req.Remove("$db")
```

Comment:
Let's at least update the function description to mention that it modifies the argument (which is not expected at all from the conversion function)